packages/tour-kit/src/components/tour-kit-spotlight.tsx

```diff
@@ -35,15 +47,30 @@ const TourKitSpotlight: React.FunctionComponent< Props > = ( { referenceElement
 		),
 	];
 
-	const { styles: popperStyles, attributes: popperAttributes } = usePopperHandler(
+	const { styles: popperStyles, attributes: popperAttributes } = usePopper(
 		referenceElement,
-		popperElementRef.current,
-		modifiers
+		popperElement,
+		{
+			strategy: 'fixed',
+			placement: 'bottom',
+			modifiers,
+		}
 	);
 
+	const clipDimensions = referenceRect
+		? {
+				width: `${ referenceRect.width }px`,
+				height: `${ referenceRect.height }px`,
+		  }
+		: null;
+
 	const clipRepositionProps = referenceElement
 		? {
-				style: popperStyles?.popper,
+				style: {
+					...( clipDimensions && clipDimensions ),
```

Comment:
Do we really need the ` clipDimensions && ` check or can we just do`...clipDimensions`? `...null` should be fine.

Reply:
Thanks @tyxla. Added follow-up issue. 🙇

Reply:
Thanks, Christos 🙌